src/DynamoCoreWpf/Views/Preview/CameraExtensions.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using HelixToolkit.Wpf.SharpDX;
+using SharpDX;
+
+namespace Dynamo.Controls
+{
+    internal static class CameraExtensions
+    {
+        private static Matrix WorldToModelMatrix()
+        {
+            return new Matrix(
+                1, 0, 0, 0,
+                0, 0, 1, 0,
+                0, -1, 0, 0,
+                0, 0, 0, 1
+                );
+        }
```

Comment:
We need this additional matrix transformation here as the model up vector being used in Dynamo is flipped wrt the default up vector in Helix.